package Egametang;

option cc_generic_services = true;

message EchoRequest
{
	required int32 num = 1;
};

message EchoResponse
{
	required int32 num = 1;
};

service EchoService
{
	rpc Echo(EchoRequest) returns (EchoResponse);
};